.gitattributes

```diff
@@ -0,0 +1,7 @@
+/.coveralls.yml export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/build.xml
+/phpunit.xml.dist
+/Tests export-ignore
```

Comment:
Is it best practise to exclude the tests from archives? I'm not sure about that.

Reply:
IMO the "archived" version of a bundle is the one that is going to be installed on prod environments via composer.
In that case the tests are not needed on "end-users" environments, as those devs are not focused on writing new tests for the library.
Tests will be available when cloning the repository, operation performed when willing to contribute to the project.

WDYT?

Reply:
For example, github.com/symfony/symfony does not exclude them. https://github.com/doctrine/common/blob/master/.gitattributes does.

I would not bother setting up `.gitattributes` myself, but I'm also fine merging this.